anta/tests/routing/bgp.py

```diff
@@ -144,6 +144,55 @@ def _add_bgp_routes_failure(
     return failure_routes
 
 
+def _check_peer_capabilities(bgp_output: dict[str, Any], capabilities: list[MultiProtocolCaps], *, strict: bool = False) -> dict[str, Any]:
```

Comment:
We need unit tests for this :)

Reply:
Removed the helper function. added check in test.